src/main/java/seedu/address/storage/JsonAdaptedPerson.java

```diff
@@ -53,10 +48,9 @@ public JsonAdaptedPerson(Person source) {
         name = source.getName().fullName;
         phone = source.getPhone().value;
         email = source.getEmail().value;
-        address = source.getAddress().value;
-        tags.addAll(source.getTags().stream()
-                .map(JsonAdaptedTag::new)
-                .collect(Collectors.toList()));
+        classGroup = source.getClassGroup().classGroup;
+        telegram = source.getTelegram().isPresent() ? source.getTelegram().get().telegramId : "";
+        github = source.getGithub().isPresent() ? source.getGithub().get().githubId : "";
     }
```

Comment:
Referring to my suggestion in Github.java and Telegram.java, this can be replaced with `orElse(Telegram.EMPTY)

Reply:
fixed!